packages/web3/package.json

```diff
@@ -52,6 +52,9 @@
         "build": "rollup -c",
         "dev": "rollup -c -w",
         "test": "jest",
+        "docker": "docker-compose run --workdir='/home/node/packages/web3' web3 npm run",
+        "predocker": "docker-compose run web3 npm install --unsafe-perm",
+        "test:integration": "npm run docker test --testMatch='/**/*.Itest.js'",
```

Comment:
do we need to introduce a new extension? Can it be done with `--scope` flag?

Reply:
I f****g hate jest and do not know what kind of magic it can do. It is very probable that there is a more elegant way.

Reply:
tell me.. I spent 10 minutes writing the actual tests / docker stuff and hours in Lerna and Jest swamp  `</rant>`